6.4 compatibility

Hi,

do you have any plans to make your plugin shopware 6.4 compatible?
I.e. i've got currently an exception when building the admistration:

ModuleNotFoundError: Module not found: Error: Can't resolve 'src/core/data-new/criteria.data' in '[...]vendor/runelaenen/shopware6-advanced-banners/src/Resources/app/administration/src/module/sw-cms/elements/rl-advanced-banner-slider/config'

--- a/src/Resources/app/administration/src/module/sw-cms/elements/rl-advanced-banner-slider/config/index.js
+++ b/src/Resources/app/administration/src/module/sw-cms/elements/rl-advanced-banner-slider/config/index.js
@@ -39,7 +39,7 @@ Component.register('sw-cms-el-config-rl-advanced-banner-slider', {
                 Context.api
             );
 
-            const criteria = new Criteria(1, 1000);
+            const criteria = new Criteria(1, 100);
             criteria.setIds(this.element.config.advancedBanners.value);
 
             this.bannerRepository

--- a/src/RuneLaenenAdvancedBanners.php
+++ b/src/RuneLaenenAdvancedBanners.php
@@ -7,8 +7,6 @@ use Shopware\Core\Framework\DataAbstractionLayer\EntityRepositoryInterface;
 use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
 use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsAnyFilter;
 use Shopware\Core\Framework\Plugin;
-use Shopware\Core\Framework\Plugin\Context\ActivateContext;
-use Shopware\Core\Framework\Plugin\Context\DeactivateContext;
 use Shopware\Core\Framework\Plugin\Context\UninstallContext;
 
 class RuneLaenenAdvancedBanners extends Plugin
@@ -33,9 +31,9 @@ class RuneLaenenAdvancedBanners extends Plugin
         // Drop tables
         /** @var Connection $connection */
         $connection = $this->container->get(Connection::class);
-        $connection->executeQuery('SET FOREIGN_KEY_CHECKS=0;');
-        $connection->executeQuery('DROP TABLE IF EXISTS `rl_ab_banner`');
-        $connection->executeQuery('DROP TABLE IF EXISTS `rl_ab_banner_translation`');
+        $connection->executeStatement('SET FOREIGN_KEY_CHECKS=0;');
+        $connection->executeStatement('DROP TABLE IF EXISTS `rl_ab_banner`');
+        $connection->executeStatement('DROP TABLE IF EXISTS `rl_ab_banner_translation`');
 
         // Remove CMS blocks & slots
         /** @var EntityRepositoryInterface $cmsBlockRepository */
@@ -66,6 +64,6 @@ class RuneLaenenAdvancedBanners extends Plugin
             $context->getContext()
         );
 
-        $connection->executeQuery('SET FOREIGN_KEY_CHECKS=1;');
+        $connection->executeStatement('SET FOREIGN_KEY_CHECKS=1;');
     }
 }

